hack/overlays/component-labels.yaml

```
@@ -0,0 +1,30 @@
+# Copyright 2021 VMware
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#@ load("@ytt:overlay", "overlay")
+
+#@ def not_matcher(marg):
+#@   def matcher(*args, **kwargs):
+#@     return not marg(*args, **kwargs)
+#@   end
+#@   return matcher
+#@ end
+
+#@overlay/match by=not_matcher(overlay.subset({"kind":"Namespace"})),expects="1+"
```

https://carvel.dev/ytt/docs/v0.40.0/lang-ref-ytt-overlay/#overlaynot_op

```suggestion
#@overlay/match by=overlay.not_op(overlay.subset({"kind":"Namespace"})),expects="1+"
```

---

awesome!